rdflib/__init__.py

```diff
@@ -45,7 +45,7 @@
 __docformat__ = "restructuredtext en"
 
 # The format of the __version__ line is matched by a regex in setup.py
-__version__ = "6.2.0-alpha"
+__version__ = "6.2.0"
 __date__ = "2021-12-20"
```

Comment:
Should this date also be updated?
```suggestion
__date__ = "2022-12-20"
```